combinations/hash.tsv

```diff
@@ -418,9 +418,8 @@ numpy=1.23.5,bacphlip=0.9.6,hmmer=3.3.2
 vcf2maf=1.6.21,ensembl-vep=109.3,samtools=1.17,bcftools=1.17
 python=2.7,deepTools=2.2.2
 r-yaml=2.3.7,r-ggplot2=3.4.2,r-tidylog=1.0.2,r-tidyverse=2.0.0,r-stringr=1.5.0,bioconductor-gsva=1.46.0,bioconductor-singscore=1.18.0,r-factominer=2.8.0
-ms2pip=3.11.0,pyopenms=2.9.1
```

Comment:
This will have no effect. So the containers will not be deleted.

Reply:
Indeed, I thought it might make sense to remove just the entries to keep the tsv as small as possible. If that is not desired, let me know

Reply:
Let keep them in for now. Thanks for caring!

Reply:
✅